test/PowerShellEditorServices.Test.Host/ServerTestsBase.cs

```diff
@@ -305,5 +303,19 @@ protected void KillService()
 
             return await requestTask;
         }
+
+        private static string GetPwshExeName()
+        {
+#if !CoreCLR
+            return "powershell.exe";
+#else
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
if you didn't have this `if` and instead just `return pwsh` does that not work?

Reply:
For another PR perhaps but should we have some form of override to allow testing against `pwsh-preview`?  Maybe an env var check or something?